rules/do-you-use-gravatar-for-your-profile-pictures/rule.md

```diff
@@ -1,29 +1,29 @@
 ---
 type: rule
-archivedreason: 
-title: Do you use Gravatar for your profile pictures?
-guid: b790dbfc-fe37-4a4a-992e-c9294620a06b
+title: Do you use Gravatar(or Cravatar for China) for your profile pictures?
```

Comment:
Hi @babakamyljanovssw 

- [ ] 1. Fix the missing space before brackets
- [ ] 2. Should we say "Cravatar in China" instead of "Cravatar for China"?  Check with Uly or other native English speaker

Reply:
I've asked Nick C, and he suggested to stay with "Cravatar for China".